docs/assets/css/highlight-truffle.css

```diff
@@ -44,7 +44,7 @@
   font-family: "Open sans";
   transform: translate(-3vw, calc(-100% + 3.05vw));
   padding: 0.5vw 1vw 3vw;
-  text-shadow: 0 -1px 1px rgba(0, 0, 0, 0.5);
+  /* text-shadow: 0 -1px 1px rgba(0, 0, 0, 0.5); */
```

Comment:
we should probs delete this I guess?
```suggestion
```

Reply:
Totally. I just commented this out initially because I was unsure if it aided readability in dark mode (I was primarily working in light mode at the time).